src/ordering/cuthillMckeeOrdering.c

```diff
@@ -0,0 +1,204 @@
+/*PGR-GNU*****************************************************************
+File: cuthillMckeeOrdering.c
+
+Generated with Template by:
+Copyright (c) 2022 pgRouting developers
+Mail: project@pgrouting.org
+
+Function's developer:
+Copyright (c) 2022 Shobhit Chaurasia
+Mail: 000shobhitchaurasia@gmail.com
+------
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ ********************************************************************PGR-GNU*/
+
+/** @file cuthillMckeeOrdering.c
+ * @brief Connecting code with postgres.
+ *
+ */
+#include <stdbool.h>
+#include "c_common/postgres_connection.h"
+#include "utils/array.h"
+
+#include "c_common/debug_macro.h"
+#include "c_common/e_report.h"
+#include "c_common/time_msg.h"
+
+#include "c_common/edges_input.h"
+#include "c_common/arrays_input.h"
+#include "c_types/ii_t_rt.h"
+
+
+#include "drivers/ordering/cuthillMckeeOrdering_driver.h"
+
+PGDLLEXPORT Datum _cuthillMckeeOrdering(PG_FUNCTION_ARGS);
```

Comment:
Do not use camelCase on the function name on the C code:

If SQL says: (Use camel case here is ok)
```
CREATE FUNCTION _pgr_cuthillMckeeOrdering(
```
PostgreSQL automatically converts everything to lower case.
`_pgr_cuthillMckeeOrdering` -> `_pgr_cuthillmckeeordering`

Then postgres will look on the C files the function that has name:  `_pgr_cuthillmckeeordering`
If a different name is used on the C file, postgres will never find it.